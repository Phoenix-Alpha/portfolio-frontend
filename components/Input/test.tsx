import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'

import { Input } from '.'

describe('<Input />', () => {
  it('should renders correct <Input />', () => {
    const wrapper = shallow(<Input />)

    expect(toJSON(wrapper)).toMatchSnapshot()
  })
})
