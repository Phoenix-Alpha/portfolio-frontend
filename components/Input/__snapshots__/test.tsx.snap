// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Input /> should renders correct <Input /> 1`] = `
<label>
  <input
    className="disabled:bg-gray-5"
  />
</label>
`;
